Server/AIServer/ServerDlg.cpp

```diff
@@ -883,15 +902,16 @@ BOOL CServerDlg::LoadNpcPosTable(std::vector<model::NpcPos*>& rows)
 
 					if (pMap == nullptr)
 					{
-						AfxMessageBox(_T("Error : CServerDlg,, Invaild zone Index!!"));
+						spdlog::error("NPC invalid zone [npcId:{}, npcZoneId:{}]", pNpc->m_sSid, pNpc->m_sCurZone);
+						AfxMessageBox(_T("NPC invalid zone index error (see log)"));
 						return FALSE;
 					}
 
 					//pNpc->Init();
 					//m_arNpc.Add(pNpc);
 					if (!m_arNpc.PutData(pNpc->m_sNid, pNpc))
 					{
-						TRACE(_T("Npc PutData Fail - %d\n"), pNpc->m_sNid);
+						spdlog::warn("Npc PutData Fail - {}", pNpc->m_sNid);
```

Comment:
This should be an error; it's *absolutely* not intended for an NPC to fail to load.
I'm pretty sure you used error for this in other cases.